### PR TITLE
Skip the role assignment when creating resources and fix the relative script path

### PR DESCRIPTION
**Pull request by michaelqi793**
Skip the role assignment when creating service principal, because service principal visiting keyvault does not need those extra roles; fix the changed relative script path

## Purpose

* ...
fix the key vault samples
## Does this introduce a breaking change?

```
[ ] Yes
[x ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[x ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

**Comment by michaelqi793** [comment]
@yiliuTo  FYI

**Review by chenrujun — DISMISSED** [review]
LGTM.

File: keyvault/azure-spring-boot-starter-keyvault-certificates/keyvault-certificates-client-side/README.md [hunk 1]
```diff
@@ -15,8 +15,9 @@ This sample illustrates how to use [Azure Spring Boot Starter Key Vault Certific
 
 This sample should work together with [azure-spring-boot-sample-keyvault-certificates-server-side].
 
-## Getting started
+This sample works for Linux/Unix OS.
```
Comment (backwind1233) [inline keyvault/azure-spring-boot-starter-keyvault-certificates/keyvault-certificates-client-side/README.md:18]: can this sample works for PC?

Comment (backwind1233) [reply keyvault/azure-spring-boot-starter-keyvault-certificates/keyvault-certificates-client-side/README.md:18]: I think you want to say, the steps in the sample  , using powershell to create resource is not working in PC‘s environment, right?

Comment (chenrujun) [reply keyvault/azure-spring-boot-starter-keyvault-certificates/keyvault-certificates-client-side/README.md:18]: Can't work for Windows power shell.

Maybe `This script can only run by bash shell` is better.

**Review by backwind1233 — APPROVED** [review]
LGTM

**Pull request merged**